docs/uds-packages/requirements/uds-package-requirements.md

```diff
@@ -139,4 +139,5 @@ Template CODEOWNERS file:
 # /* @name-of-established-known-team #optional addition to "uds-package-maintainers"
 
 /CODEOWNERS @defenseunicorns/uds-package-maintainers
```

Comment:
```suggestion
/CODEOWNERS @jeff-mccoy @daveworth 
```
Is this supposed to be this? (not reqing changes bc I am not sure)

Reply:
I believe @Racer159 is correct here in his suggestion

Reply:
fine with me :) - this is an example CODEOWNERS to be used in all uds-package-*

will also need to update in uds-package-template https://github.com/defenseunicorns/uds-package-template/blob/main/CODEOWNERS